react/craco.config.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+
+module.exports = {
+  webpack: {
+    alias: {
+      '@': path.resolve(__dirname, 'src/')
```

Comment:
Czy to dobrze działa z zagnieżdżonymi plikami, np. `import { cosTam } from '@/cosTam/jeszczeCosTam/plik'`? Jeśli nie, to spróbowałbym zrobić coś takiego:

`'@/*': path.resolve(__dirname, 'src/*')`

Reply:
Właśnie o dziwo działało, ale wolę nie kusić losu, dlatego zrobię to _po bożemu_ (zgodnie z dokumentacją — stosując zapis `@/*` właśnie).